waspc/cli/src/Wasp/Cli/Command/Test.hs

```diff
@@ -39,7 +42,7 @@ watchAndTest testRunner = do
   watchOrStartResult <- liftIO $ do
     ongoingCompilationResultMVar <- newMVar (warnings, [])
     let watchWaspProjectSource = watch waspRoot outDir ongoingCompilationResultMVar
-    watchWaspProjectSource `race` testRunner outDir
+    watchWaspProjectSource `race` testRunner waspRoot
```

Comment:
Instead of running from the `.wasp/out` dir, it now runs from the project root dir.